Backend/Remora.Discord.API/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -92,18 +105,9 @@ public static IServiceCollection AddDiscordApi
                         .AddStickerObjectConverters();
 
                     options.AddDataObjectConverter<IUnknownEvent, UnknownEvent>();
+                    options.AddConverter<PropertyErrorDetailsConverter>();
 
-                    options
-                        .AddConverter<OptionalConverterFactory>()
-                        .AddConverter<NullableConverterFactory>()
-                        .AddConverter<SnowflakeConverter>()
-                        .AddConverter<ColorConverter>()
```

Comment:
Is Discord's color implementation standard enough to where this was moved out to Remora.Rest? I thought they had homebrewed something.

Reply:
Yeah, it's just the RGB triplet in integer form. It's nothing special.